ios/edge/AppDelegate.mm

```diff
@@ -8,6 +8,8 @@
 #import "ExpoModulesCore-Swift.h"
 #import "Edge-Swift.h"
 #import "RNBootSplash.h"
+#import <AdServices/AdServices.h>
+#import <AppTrackingTransparency/AppTrackingTransparency.h>
```

Comment:
Adding a header file does nothing, unless you are using the code inside. It's basically the same as an `import` statement in JavaScript.

Ok, maybe some `import` statements have side effects, and I guess maybe that's true here too, but it's pretty weird for a C-style language. I'm guessing you can delete these.